packages/calypso-e2e/src/lib/pages/posts-page.ts

```diff
@@ -101,8 +106,21 @@ export class PostsPage {
 	 * Clicks on the `add new post` button.
 	 */
 	async newPost(): Promise< void > {
-		const locator = this.page.locator( selectors.addNewPostButton );
-		await locator.click();
+		const newPostLocator = this.page.locator( selectors.addNewPostButton );
+
+		// Get ready for some temporary yuckiness!
+		// On atomic sites, the Gutenframe does not play well with our E2E tests.
+		// For the time being, we need to make redirect any click navigations from this Calypso page
+		// to the wp-admin version of the editor.
```

Comment:
I like that it's succinct and this is likely the least yucky we can make it. Nice work.